keyboards/tssouthpaw/keyboard.json

```diff
@@ -0,0 +1,335 @@
+{
+    "manufacturer": "TS Design Works LLC",
+    "keyboard_name": "TS-Southpaw-Rev-1.6",
+    "maintainer": "ts_design_works_llc",
+    "bootloader": "rp2040",
+    "processor": "RP2040",
+    "diode_direction": "COL2ROW",
+    "url": "https://github.com/tsdesignworks/tssouthpaw",
+    "build": {
+        "lto": true
+    },
+    "matrix_pins": {
+        "cols": ["GP26", "GP25", "GP24", "GP23", "GP22", "GP16", "GP17", "GP15", "GP14", "GP13", "GP12", "GP11", "GP10", "GP9", "GP8", "GP7", "GP6", "GP5", "GP4", "GP3", "GP2"],
+        "rows": ["GP29", "GP28", "GP21", "GP20", "GP19", "GP18"]
+    },
+    "usb": {
+        "device_version": "0.0.2",
+        "pid": "0x6163",
+        "vid": "0x7856",
+        "suspend_wakeup_delay": 200
+    },
+    "ws2812": {
+    "driver": "vendor",
+    "pin": "GP27"
+    },
+    "encoder": {
+        "enabled": true,
+        "rotary": [
+            {"pin_a": "GP1", "pin_b": "GP0", "resolution": 4}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "caps_word": true,
+        "extrakey": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "caps_word": {
+        "enabled": true
+    },
+    "debounce": 5,
+    "rgb_matrix": {
+        "driver": "ws2812",
+        "led_count": 104,
+        "timeout": 600000,
+        "sleep": true,
+        "led_process_limit": 8,
+        "led_flush_limit": 16,
+        "max_brightness": 250,
```

Comment:
This is pretty much the max (255), so I'm not sure what setting it to this value would actually gain you. 

```suggestion
```